themes/default/content/blog/kubernetes-fundamentals-part-two/index.md

```diff
@@ -0,0 +1,215 @@
+---
+title: "Kubernetes Fundamentals Part Two"
+date: 2021-09-02T12:54:53-05:00
+draft: false
+meta_image: k8s-fundamentals.png
+meta_desc: "Kubernetes is everywhere now, but it’s largely been the domain of people who work on the Ops side of things. What about devs, though?"
+authors:
+    - kat-cosgrove
+tags:
+    - kubernetes
+    - google-cloud
+---
+Kubernetes is everywhere now, but it’s primarily been the domain of people working on the Ops side of infrastructure. What about devs, though? You benefit from knowing what Kubernetes is and how to use it, too -- otherwise, we’re still putting teams in silos. In this blog, we're going to build off part one by learning about managed kubernetes services: what they are, when they're useful, and how you can try deploying to one yourself, starting with Google's GKE.
+<!--more-->
+
+## What is a managed Kubernetes service?
+
+Essentially, this just means that someone else is responsible for some of the work involved in standing up and maintaining Kubernetes. It varies from provider to provider, but services can be pretty bare-bones or include everything under the sun, handing you a pre-configured environment and a dedicated support team to help you.
+
+## What makes this useful?
+
+Kubernetes is very powerful, but also very complicated. The flexibility of the platform means that there's a lot of configuration you have to do just to get off the ground, and a managed service takes some of that off your plate. Some configurations are abstracted away, and some decisions are made for you. Things you would otherwise have to set up yourself, like load balancing, monitoring, and updates are covered by the vendor. It's less effort to get started, and less effort to maintain. Scaling is painless, since you can just provision more resources. For most people, a managed service is flat-out more reliable.
```

Comment:
```suggestion
Kubernetes is very powerful, but also very complicated. The flexibility of the platform means that there's a lot of configuration you have to do just to get off the ground, and a managed service takes some of that off your plate. Some configurations are abstracted away, and some decisions are made for you. Things you would otherwise have to set up yourself, like load balancing, monitoring, and updates, are covered by the vendor. It's less effort to get started and less effort to maintain. Scaling is painless since you can just provision more resources. For most people, a managed service is flat-out more reliable.
```